templates/vue/src/components/TapestryMedia.vue

```diff
@@ -147,7 +161,15 @@ export default {
   border-radius: 15px;
   overflow: scroll;
   height: 100%;
+  padding: 0;
+
+  &-no-scroll {
+    overflow: hidden;
+  }
+}
 
+.media-wrapper-embed {
+  background: white;
```

Comment:
Please remove this and move the `&-embed` below under the above directive like it was before. I'm assuming there is no `media-wrapper-embed-embed` class, which is what is implied here.